Foolprofs install instructions (#1132)
pinatacolada/topskylppc: index.md

     ![Autoload](/assets/img/ta1.png)

> 1.   Fill out the CID and password fields with your VATSIM credentials, select your appropriate hardware devices, and at least a PTT 1 key. The remaining settings may be set according to your preference, but it is recomended to be set as per below.

Reviewer: o proibir left ctrl e absolutamente intencional, para nao ter a freq constantemente a clickar quando se esta a fazer copy paste ou outros shortcuts

     ![Autoload](/assets/img/vacs1.png)

1.   Set `MODE` to `Voice activation` , then click X to close the `Transmit Config` window.

     ![Autoload](/assets/img/vacs2.png)

1.   Click `Hotkeys`.

     ![Autoload](/assets/img/vacs3.png)

> 1.   We recommend to set a single key for `Accept first call` and `End active call` (must not be the same key you have for PTT in TrackAudio), then click X to close the `Hotkeys Config` window.

Reviewer: ```suggestion
1.   We recommend to set the same key for `Accept first call` and `End active call` (must be different than TrackAudio PTT), then click X to close the `Hotkeys Config` window.
```
